platform-enterprise_versioned_docs/version-25.1/enterprise/studios.mdx

```diff
@@ -0,0 +1,251 @@
+---
+title: "Studios deployment"
+description: Deploy Seqera Platform with Studios
+date: "17 Mar 2025"
+tags: [docker, compose, kubernetes, studios, deployment]
+---
+
+Enable Studios as part of your Seqera Platform instance. You must have Data Explorer enabled to use Studios. Only the Amazon AWS public cloud is currently supported.
+
+:::caution 
+You must complete the upgrade of your Platform Enterprise installation to version 24.2 before enabling and configuring Studios.
+:::
+
+## DNS configuration
+
+Each Studio is reachable at a unique URL that includes a randomly generated subdomain name. For example: `https://abcd.example.com/`, where `example.com` is your Seqera base domain name.
+
+Provide a wildcard TLS certificate to allow for uniquely generated subdomains. A wildcard certificate common name includes `*.` in the domain name, such as `*.example.com`, thereby securing any subdomain name at this level.
+
+Studios uses the following set of domains and subdomains:
+
+- The domain that you set for `TOWER_SERVER_URL`, such as `example.com`.
+- A wildcard subdomain that you must configure specifically for Studios. This wildcard subdomain is the parent for each unique session URL, such as `abcd.example.com`.
+- The connection proxy, defined by `CONNECT_PROXY_URL`. This URL is a first-level subdomain of your `TOWER_SERVER_URL`. For example, `https://connect.example.com`. 
+
+## Studios workspace availability
+
+You can configure which organizational workspaces have access to Studios. This configuration is set in the `tower.yml` file. The `tower.data-studio.allowed-workspaces` field supports the following options:
+
+- `allowed-workspaces: []`: Disables Studios. This is the default if the `allowed-workspaces` field is not specified.
+- `allowed-workspaces: [ <WORKSPACE_ID>,<WORKSPACE_ID> ]`: Enables Studios for the comma-separated list of organizational workspace IDs.
+- `allowed-workspaces: null`: Enables Studios for all organizational workspaces.
```

Comment:
Non-blocking comment: the way this config param works is counter-intuitive.

Reply:
Not sure I agree - I believe it matches other config options. Can you explain why you think it's counter-intuitive?